README.md

```
@@ -8,13 +8,15 @@
 
 ## 팀원 구성
 
-| 프로필 |                    이름                     |    역할    | 담당 기능                                                         |
+## 팀원 구성
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)